src/megaapi.cpp

```diff
@@ -1370,17 +1377,17 @@ const char* MegaError::getErrorString(int errorCode, ErrorContexts context)
 
 const char* MegaError::toString() const 
 { 
-	return getErrorString(); 
+    return nullptr;
```

Comment:
probably we should cover the to string cases for  MetaTransfer::getLastError also

Reply:
Agree, well spotted!!

Reply:
Solved